pkg/controllers/state/cluster.go

```diff
@@ -67,7 +71,16 @@ type Cluster struct {
 	antiAffinityPods sync.Map // pod namespaced name -> *v1.Pod of pods that have required anti affinities
 }
 
+const (
+	// AbnormalClusterSyncDuration is the duration after which the cluster state is considered to be out of sync for a long enough time to alarm the customer
+	AbnormalClusterSyncDuration = 15 * time.Second
+	// AbnormalClusterSyncLoggingInterval is the interval at which we will log about the cluster being out of sync if it is out of sync
+	AbnormalClusterSyncLoggingInterval = 5 * time.Minute
```

Comment:
Is five minute expiry on the cache entries too long? Should I log every minute instead?

Reply:
Depending on the interval that we pick above, I think that's going to determine the interval here. If we are saying that 1m of NotSynced is a problem (I'm not sure that this is the right value, but this is probably what I would pick off the top of my head), then logging every 1m or so after it happens is probably sufficient

Reply:
I am thinking 1m instead as well.